user/list.md

```diff
@@ -0,0 +1,39 @@
+---
+description: How to list your company's users
+---
+
+# List users
```

Comment:
page title still reads "List your company's users"

Reply:
same header level 1 to title issue